stable/traefik/templates/configmap.yaml

```diff
@@ -34,6 +34,12 @@ data:
           KeyFile = "/ssl/tls.key"
       {{- end }}
     [kubernetes]
+    {{- if .Values.kubernetes.namespaces }}
+    namespaces = {{ .Values.kubernetes.namespaces }}
+    {{- end}}
+    {{- if .Values.kubernetes.labelSelector }}
+    labelselector = {{ .Values.kubernetes.labelSelector | quote }}
+    {{- end}}
```

Comment:
It would be a good idea to wrap everything from lines 37 to 42 in:

```
{{- if .Values.kubernetes.namespaces }}
{{- end}}
```

This is necessary to accommodate the scenario where `values.yaml` either does not containe `kubernetes` or contains something like this:

```
kubernetes:
  # namespaces: ~
  # labelSelector: ~
```

In the former case, `.Values.kubernetes` evaluates as `nil` and template processing will fail because a `nil` doesn't have a `namespaces` or `labelSelector` field. In the latter case (the sample above), `kubernetes` evaluates as an empty string and will have the same problem-- neither of those two fields will exist.

We learned this while working on / evaluating #414.